ledger-core/virtual/integration/deduplication/find_call_request_handler_test.go

```diff
@@ -482,7 +482,9 @@ func (s *VFindCallRequestHandlingSuite) addPayloadAndWaitIdle(
 	s.server.WaitActiveThenIdleConveyor()
 }
 
-func (s *VFindCallRequestHandlingSuite) finish() {
+func (s *VFindCallRequestHandlingSuite) finish(t *testing.T) {
+	testutils.WaitSignalsTimed(t, 10*time.Second, s.server.Journal.WaitAllAsyncCallsDone())
+	s.server.WaitIdleConveyor()
```

Comment:
Why that line is needed?

Reply:
because we need to finish all the sms that were waiting for async calls to finish their jobs

Reply:
switched to waiting of StateReport message